dcos_test_utils/dcos_api.py

```diff
@@ -361,6 +361,21 @@ def _wait_for_metronome(self):
         assert r.status_code == 200, "Expecting status code 200 for Metronome but got {} with body {}"\
             .format(r.status_code, r.content)
 
+    @retrying.retry(wait_fixed=2000,
+                    retry_on_result=lambda r: r is False,
+                    retry_on_exception=lambda _: False)
+    def _wait_for_all_healthy_services(self):
+        r = self.health.get('units')
+        r.raise_for_status()
+
+        all_healthy = True
+        for unit in r.json()['units']:
+            if unit['health'] != 0:
+                log.info("{} service health: {}".format(unit['id'], unit['health']))
+                all_healthy = False
```

Comment:
Could put a `break` here

Reply:
True... `break` would interrupt the loop at the first "unhealthy" service; I preferred as-is so that all services returning non-zero would be logged here.